Picard-public.iml

```diff
@@ -50,5 +50,4 @@
       </library>
     </orderEntry>
   </component>
-</module>
-
+</module>
```

Comment:
@vdauwera  
I did the above.  However, in IntelliJ it says project SDK is not defined.  Shouldn't I setup the SDK?